ResearchSuite/ResearchSuite/RSDOverviewStepObject.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  RSDOverviewStepObject.swift
+//  ResearchSuite
+//
+//  Copyright © 2017-2018 Sage Bionetworks. All rights reserved.
```

Comment:
This file appears to be all-new--should it just be © 2018? Or is this some weird GitHub diff glitch